pkg/dashboards/dashboards.go

```diff
@@ -36,6 +36,7 @@ import (
 )
 
 var defaultDashboardIds map[string]struct{}
+var defaultDashboardNames map[string]bool
```

Comment:
Since you're only ever storing `true` as the value in your new map, you should change `bool` to `struct{}`.